internal/api/device.go

```diff
@@ -32,6 +32,7 @@ type Device struct {
 	InterfaceCount       *int                   `json:"interfaceCount,omitempty"`
 	Core                 *DeviceCoreInformation `json:"core,omitempty"`
 	DeviceManagementType *string                `json:"deviceManagementType,omitempty"`
+	SshInterfaceId       *string                `json:"sshInterfaceId,omitempty"`
```

Comment:
According to the spec - this looks like a `*int32`.

https://developer.equinix.com/docs/launch-self-configured-device
https://developer.equinix.com/sites/default/files/ne-v1-catalog-ne_v1_0.zip

Reply:
Hi Marques, this field is a string in API. For some reasons, it is an integer in the documentation. We are going to make the changes in swagger. In the payload of the documentation(the first link in your response), it's actually a string instead of integer. We will ask team to update it to be string.

Reply:
In the spec, this is currently a value such as: `4`.  Why should it be a string? It seems an int would be the appropriate type unless other formats can be returned or set.

```
      sshInterfaceId:
        type: integer
        example: 4
        description: You may specify any available interface on the device as the sshInterfaceId. This field is only applicable to self-configured devices.
```

Reply:
We are updating the specs to change it to string.  There is a possibility of sub-interfaces. In that case, we might use '-' or '.' to represent    interface id